### ISSUE TITLE
High tier boss idea

### ISSUE THREAD

**Issue by Sixtrings** [AngelArenaAllstars/suggestions#8]
Another boss idea, potentially high tiered: Demon of Deception. The fight starts out with the boss standing in the middle of the arena, with an aura that steals away stats and auto attacks. If you hit boss with any form of damage (radiance included), it will kinda manta style and spawn two illusions along. However you have to hit the real boss 4 times total in order to be able to deal damage to it, but if you hit an illusion, it will also spawn two other illusions for a total of three. The illusions still have the auto attack animation, but do no damage whatsoever. The purpose is to figure out which one is the real boss while taking as little time as possible. For every illusion hit during this phase, the boss will gain a stackable buff that grants armor and magic resist. The stat steals from the initial encounter and the protected buff persists until the challenge is over. So that the challengers should try to pass this phase as quickly as possible while not using cleave items or of the sort.
Phase 1: The demon fires out spells in a random direction, all obvious ones, but the particular spells are decided from the heroes that step into the battle. (if a pudge is fighting then there would be pudge hooks, etc). The spell chosen would not be random, (unless programmers want to code a projectile spell for every heroe's ability for the boss to use), and the number of spells would increase as the battle drags on. The catch is that as the fight goes on, the boss will start throwing out illusion spells. Illusion spells are spells that only have the particle animation, but do no damage and inflict no status effects. The idea is that the screen is filled with spells, both real and fake, and for heroes that lack the movement speed to dodge them all, they would have to risk the probability of a fake spell.
Phase 2: After a percentage of hp is removed, the boss throws out target point projectiles called Fracturing. The hero hit with this projectile will have one of its abilities muted for the rest of the battle, and spawn an illusion similar to Morphling's ult (the illusion does not die when the hero dies and the illusion has access to only the spell that was muted). Both the original hero and the clone will have half the stats of the orginal. If a hero is hit with another one of these Fracturing spells while under the effect of one, then an additional illusion is spawned and the stats of the illusions and original hero is reduced to a third of what it should be. Micro-management is the key to this stage, and the crowdiness serve to hinder dodging.
If an Fractured illusion dies during the battle, the hero loses that ability entirely until the battle is over. If the original hero dies while saving an illusion, the Hero will not be considered dead. They would respawn at the location of the Fractured illusion.
Jack Grayling - Today at 11:02 PM
Only heroes will be targetted by the Fracturing spell, Heroes hit with three Fracturing spells (split into four) will no longer be able to be hit by the spell again
As of currently, there is no phase 3, as I believe it is difficult enough. This could still be changed

 tl;dr Phase 1 is whack-a-mole (Lasts 4 hits), Phase 2 is dodge the projectiles-but-it-might-be-fake, Phase 3 is Phase 2+ your abilities are now on illusions

**Comment by Haganeko** [comment]
This seems like a lot of busy work. Phase three has served as inspiration for Stop Killing Yourself.
